### DISCUSSION TITLE
How to configure parcel to support Web Components in Vue 3?

### DISCUSSION THREAD

**Discussion by TobiasSchikora** [parcel-bundler/parcel#8822] [Q&A] [answered]
Hallo!

We tried to implement a custom element in our vue app. We got the following message:

```
The `compilerOptions` config option is only respected when using a build of Vue.js that includes the runtime compiler (aka "full build"). Since you are using the runtime-only build, `compilerOptions` must be passed to `@vue/compiler-dom` in the build setup instead.
- For vue-loader: pass it via vue-loader's `compilerOptions` loader option.
- For vue-cli: see https://cli.vuejs.org/guide/webpack.html#modifying-options-of-a-loader
- For vite: pass it via @vitejs/plugin-vue options. See https://github.com/vitejs/vite/tree/main/packages/plugin-vue#example-for-passing-options-to-vuecompiler-dom
```

I did not find an answer how to configure **isCustomElement** in parcel 2 (vue 3). I found this: https://github.com/parcel-bundler/parcel/issues/8030 but I don’t know where to configure it.

Can anybody please help to get it run? Thank you very much!

Greets,
Tobias

**Comment by mischnic** [comment, marked as answer]
One of these files:
https://github.com/parcel-bundler/parcel/blob/adb01fe60c10dc119cffcd025f8c30a332547793/packages/transformers/vue/src/VueTransformer.js#L26-L33
containing e.g.
```
{
	"compilerOptions": ....
}
```
(or `module.exports = { ... }`)

**Reply by TobiasSchikora** [reply]
It works! Thank you so much, @mischnic !
